components/mate/caja/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Till Wegmueller
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		caja
+COMPONENT_VERSION=	1.12.6
+COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
+COMPONENT_SUMMARY=	the file manager for the MATE desktop
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:09516e2a781a2c1574a28c7d9e3a6720cd7af2fbe7f7c0b7b9891923d2f31ac1
+COMPONENT_ARCHIVE_URL=	https://github.com/mate-desktop/$(COMPONENT_NAME)/archive/v$(COMPONENT_VERSION).tar.gz
+COMPONENT_LICENSE= GPLv2, LGPLv2, FDLv1.1
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION = System/Libraries
+COMPONENT_FMRI = desktop/mate/$(COMPONENT_NAME)
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+COMPONENT_PREP_ACTION =        ( cd $(@D)  && \
+					./autogen.sh --help )
+
+
+CONFIGURE_OPTIONS+=	--sysconfdir=/etc
+CONFIGURE_OPTIONS+=	--disable-static
+CONFIGURE_OPTIONS+=	--disable-scrollkeeper
+CONFIGURE_OPTIONS+=	--localstatedir=/var/lib
+CONFIGURE_OPTIONS+=	--with-gnome-distributor="The OpenIndiana Project"
+CONFIGURE_OPTIONS+=     --libexecdir=/usr/lib/$(COMPONENT_NAME)
```

Comment:
This should be just /usr/lib for 32 bit files and /usr/lib/$(MACH64) for 64-bit. You can use $(CONFIGURE_LIBDIR.$(BITS))